flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -149,7 +150,14 @@ class FlxPexParser
 			maxColor: FlxColor.fromRGBFloat(minR + varR, minG + varG, minB + varB, minA + varA)
 		};
 	}
-	
+
+	private static inline function getNodeName(property:String, config:Fast):String
+	{
+		// for backwards compatibility, check for versions of properties that
```

Comment:
Oh, so this is an issue for all properties? I thought it was specifically `"finishParticleSizeVariance"`.